applications.yaml

```
@@ -251,6 +251,12 @@
     id: Everything.exe
   options:
   - tray_and_multi_window
+- name: Everything1.5a
+  identifier:
+    kind: Exe
+    id: Everything64.exe
+  options:
+  - tray_and_multi_window
```

```suggestion
    kind: Class
    id: EVERYTHING_(1.5a)
  options:
  - force
  - tray_and_multi_window
```

It is preferred to use window classes over process names if possible.